app/src/main/java/com/globant/example/mentorapp/data/entity/UserEntity.java

```diff
@@ -0,0 +1,282 @@
+
+package com.globant.example.mentorapp.data.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class UserEntity {
+
+    @SerializedName("login")
```

Comment:
If the attribute name matches the serialized name is not necessary to define this annotation